packages/cmf/src/sagaRouter/index.js

```diff
@@ -1,7 +1,8 @@
-import { all, fork } from 'redux-saga/effects';
+import { all, fork, call } from 'redux-saga/effects';
 import { browserHistory as history } from 'react-router';
 import routerSaga from './router';
+import changeDocumentTitle from './documentTitle';
 
 export default function* initSagaRouter(routes) {
-	yield all([fork(routerSaga, history, routes)]);
+	yield all([fork(routerSaga, history, routes), call(changeDocumentTitle)]);
```

Comment:
I think this shouldn't be activated by default but exported to be available in host application